x-pack/test_serverless/api_integration/test_suites/common/elasticsearch_api/home.ts

```diff
@@ -6,14 +6,28 @@
  */
 
 import { FtrProviderContext } from '../../../ftr_provider_context';
-
+import { RoleCredentials } from '../../../../shared/services';
 export default function ({ getService }: FtrProviderContext) {
-  const esSupertest = getService('esSupertest');
+  const supertestWithoutAuth = getService('supertestWithoutAuth');
+
   const svlCommonApi = getService('svlCommonApi');
+  const svlUserManager = getService('svlUserManager');
+  let roleAuthc: RoleCredentials;
 
   describe('Home', function () {
+    before(async () => {
+      roleAuthc = await svlUserManager.createApiKeyForRole('admin');
```

Comment:
_question_: will this test fail with `viewer` role?

Reply:
yes